src/features/auth/forgottenPassword/ForgottenPassword.tsx

```diff
@@ -26,7 +27,20 @@ export const ForgottenPassword: FunctionComponent = () => {
   const shouldDisableValidateButton = isValueEmpty(email)
 
   const { navigate } = useNavigation<UseNavigationType>()
-
+  const { refetch, isFetching } = useQuery('forgottenPassword', forgottenPasswordQuery, {
+    cacheTime: 0,
+    enabled: false,
+    onSuccess: () => {
+      navigate('ResetPasswordEmailSent', { email })
+    },
+  })
+  async function forgottenPasswordQuery() {
+    try {
+      await api.postnativev1requestPasswordReset({ email })
+    } catch (err) {
+      throw new AsyncError('NETWORK_REQUEST_FAILED', refetch)
+    }
+  }
   function onBackNavigation() {
```

Comment:
Nitpick : les sauts de lignes entre déclarations de fonctions
```suggestion
  })
  
  async function forgottenPasswordQuery() {
    try {
      await api.postnativev1requestPasswordReset({ email })
    } catch (err) {
      throw new AsyncError('NETWORK_REQUEST_FAILED', refetch)
    }
  }
  
  function onBackNavigation() {
```